ciao-controller/api/api.go

```diff
@@ -440,12 +440,29 @@ func addWorkload(c *Context, w http.ResponseWriter, r *http.Request) (Response,
 		return errorResponse(err), err
 	}
 
+	// we allow admin to create public workloads for any tenant. However,
+	// users scoped to a particular tenant may only create workloads
+	// for their own tenant.
+	vars := mux.Vars(r)
+	tenantID, ok := vars["tenant"]
+	if ok {
+		req.TenantID = tenantID
+	} else {
+		req.TenantID = "public"
```

Comment:
@kaccardi for image service, I'm planning to use the ``visibility`` metadata field for specifiying if the image is public, internal or private. A similar approach as the used at: https://wiki.openstack.org/wiki/Glance-v2-community-image-visibility-design